ContactManger/src/main/java/Manager.java

```diff
@@ -0,0 +1,45 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class Manager {
+    private List<Contact> myFriends;
+    int friendsCount;
+
+    public Manager() {
+        friendsCount = 0;
+        myFriends = new ArrayList<>();
+    }
+
+    public void addContact(Contact contact) {
+        myFriends.add(contact);
+        friendsCount++;
+    }
+
+    public List<Contact> searchContact(String name) {
+        List<Contact> searchResult = new ArrayList<>();
+        for (Contact customer : myFriends) {
+            if (customer.getName().equals(name)) {
+                searchResult.add(customer);
+            }
+        }
+        return searchResult;
+    }
+
+    public void showContacts() {
```

Comment:
@anamariaTapia This method is not related to the class, this functionality seems more from the console than the Manager class